Source/JavaScriptCore/wasm/WasmB3IRGenerator.cpp

```diff
@@ -787,21 +798,24 @@ class B3IRGenerator {
         return m_outerLoops.last();
     }
 
-    ExpressionType push(Type type)
-    {
-        return push(toB3Type(type));
-    }
-
-    ExpressionType push(B3::Type type)
+    ExpressionType makePushVariable(B3::Type type)
```

Comment:
I wonder if we should just name it `getPushVariable`, or `getStackVariable` since the purpose of this function is not making it: if it already exists, we just return it without "making" it. So calling it `make` seems wrong to me.